puppet/modules/web/manifests/init.pp

```diff
@@ -15,37 +15,6 @@
     letsencrypt => $https,
```

Comment:
I'd prefer to drop this parameter and move the inclusion into the vhost:
https://github.com/theforeman/foreman-infra/blob/cc1300e15cbb07e3b94ded6aaa53f0fbe77132fe/puppet/modules/web/manifests/base.pp#L8-L10

Reply:
That will break using `web::base` with HTTPS directly, like it's done by `redmine` and `profiles::jenkins::controller`, as those do not use `web::vhost`.

Reply:
Can we modify those to also include the base? I think b31f6de8158d629c1ba7751a84a07b2db5650789 should work and makes it all more consistent.